atom/browser/api/atom_api_app.cc

```diff
@@ -370,6 +376,8 @@ mate::ObjectTemplateBuilder App::GetObjectTemplateBuilder(
 #if defined(OS_MACOSX)
       .SetMethod("hide", base::Bind(&Browser::Hide, browser))
       .SetMethod("show", base::Bind(&Browser::Show, browser))
+      .SetMethod("isDarkModeEnabled",
```

Comment:
Probably just `isDarkMode`, since it returns whether system _is_ in dark mode, instead of whether system _can_ use dark mode.